src/gql/generated/types.ts

```diff
@@ -2134,6 +2136,23 @@ export type VersionTaskStatusStatsArgs = {
   options?: InputMaybe<BuildVariantOptions>;
 };
 
+/** Version models a commit within a project. */
+export type VersionVersionTasksArgs = {
```

Comment:
supports my argument for using version.Tasks instead. This would become `VersionTasksArgs` instead.